yt_dlp/networking/websocket.py

```diff
@@ -2,7 +2,7 @@
 
 import abc
 
-from .common import Response, RequestHandler
+from .common import RequestHandler, Response
```

Comment:
```suggestion
from .common import Response, RequestHandler
```

revert, fixed in cleanup